google-cloud-dialogflow-cx-v3/.repo-metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "api_id": "dialogflow.googleapis.com",
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-dialogflow-cx-v3/latest",
+    "distribution_name": "google-cloud-dialogflow-cx-v3",
+    "language": "ruby",
+    "name": "dialogflow",
+    "name_pretty": "Dialogflow CX V3 API",
+    "product_documentation": "https://cloud.google.com/dialogflow",
+    "repo": "googleapis/google-cloud-ruby",
+    "requires_billing": true,
+    "ruby-cloud-description": "Dialogflow is an end-to-end, build-once deploy-everywhere development suite for creating conversational interfaces for websites, mobile applications, popular messaging platforms, and IoT devices. You can use it to build interfaces (such as chatbots and conversational IVR) that enable natural and rich interactions between your users and your business. Note that google-cloud-dialogflow-cx-v3 is a version-specific client library. For most uses, we recommend installing the main client library google-cloud-dialogflow-cx instead. See the readme for more details.",
```

Comment:
It's too bad this description doesn't explain CX (vs ES), but I suppose anyone using it will know what it is.

Reply:
It's a good point, though. I'll open a work item (to both clients) to note the distinction.